### ISSUE TITLE
Reference to PeterO.Cbor version which contains a public advisory

### ISSUE THREAD

**Issue by timesscar** [MichaelGrafnetter/DSInternals#140]
Hello,

During a review of the DsInternals code, [this advisory](https://github.com/advisories/GHSA-6r92-cgxc-r5fg) was found.  The relevant commit that addresses it is [here ](https://github.com/peteroupc/CBOR/commit/b4117dbbb4cd5a4a963f9d0c9aa132f033e15b95)and is found in versions > 4.5 of the nuget.

Please advise if you would prefer for me to submit a PR with a nuget version bump or otherwise.

Thanks!

Matt

**Comment by timesscar** [comment]
I just noticed that dependabot automatically submitted a PR for the fix, https://github.com/MichaelGrafnetter/DSInternals/pull/138 for reference.

**Comment by MichaelGrafnetter** [comment]
Yes, upgrading to a new version of the library is in my TODO list. On the other hand, it is currently only used for parsing values generated by Microsoft's code in Azure AD, so I don't expect any malicious intent there.
